exporter/collector/integrationtests.md

```diff
@@ -56,17 +62,20 @@ go test -tags=integrationtest -run=TestIntegration
 
 ## Adding New Tests
 
-To add a new test:
+To add a new test, using metrics as an example:
 
-1. Create an OTLP fixture. Currently, the tests only support metrics fixtures which should be
-    JSON encoded OTLP
-    [`ExportMetricsServiceRequest`](https://github.com/open-telemetry/opentelemetry-proto/blob/b43e9b18b76abf3ee040164b55b9c355217151f3/opentelemetry/proto/collector/metrics/v1/metrics_service.proto#L35)
-    message. Put the fixture in the [`testdata/fixtures`](integrationtest/testdata/fixtures) directory. As an example,
-    see
+1. Create an OTLP fixture. The test fixtures should be JSON encoded OTLP messages of type:
```

Comment:
Nit: should we use permalinks for protos here ?

Reply:
There are permalinks for tag v0.19.0